bindings/python/examples/client/09_transaction.py

```diff
@@ -15,11 +15,11 @@
 secret_manager = MnemonicSecretManager(
     os.environ['NON_SECURE_USE_OF_DEVELOPMENT_MNEMONIC_1'])
 
-output = SendAmountParams(
+params = SendParams(
     address='rms1qzpf0tzpf8yqej5zyhjl9k3km7y6j0xjnxxh7m2g3jtj2z5grej67sl6l46',
     amount=1000000,
 )
 
 # Create and post a block with a transaction
-block = client.build_and_post_block(secret_manager, output=output)
+block = client.build_and_post_block(secret_manager, output=params)
```

Comment:
output=params is a bit weird

Reply:
true ... had the same feeling.

Reply:
I changed the function argument name to `params` as well.